h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoost.java

```diff
@@ -774,11 +778,11 @@ private boolean doScoring(final XGBoostModel model, final XGBoostExecutor exec,
     }
   }
 
-  static CustomMetric toCustomMetric(EvalMetric evalMetric) {
+  static CustomMetric toCustomMetric(EvalMetric evalMetric, boolean isTrain) {
```

Comment:
The boolean parameter looks like a code smell to me. A reader has to switch to implementation to see what's going on. IMHO, two methods like toCustomTrainingMetric() and toCustomValidationMetric() would be much more expressive.

Reply:
sounds good

Reply:
add, code looks better now - thx